omnibus/config/software/postgresql13.rb

```diff
@@ -55,9 +55,5 @@
     Dir.glob("#{install_dir}/embedded/postgresql/#{short_version}/bin/*").sort.each do |bin|
       link bin, "#{install_dir}/embedded/bin/#{File.basename(bin)}"
     end
-
-    delete "#{install_dir}/embedded/postgresql/#{short_version}/share/doc"
```

Comment:
To keep the package size down it might be good to verify that the docs are not showing up somewhere else.

Reply:
There's a very real chance we need to do similar cleanup to 13

Reply:
added the similar change for cleanup docs